### PR TITLE
Feature/player interaction

### PR DESCRIPTION
**Pull request by Jokubas126**
Initial mechanics for player movement and object pick up -> release

### CODE REVIEW

File: Assets/Scripts/HoldObject.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        if (Input.GetKeyDown("e") && pickedUpObject == null)
+        {
+            PickUp();
+        }
+
+        if (pickedUpObject != null)
+        {
+            CarryObject();
+
+            if (Input.GetKey("r"))
```
Comment (latedude2) [inline Assets/Scripts/HoldObject.cs]: we should use the same key/button for picking up and dropping

File: Assets/Scripts/HoldObject.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            {
+                Release();
+            }
+        }
+    }
+
+    private void PickUp()
+    {
+        if (Physics.Raycast(transform.position, transform.forward, out RaycastHit hit, interactionDistance, 1 << 8))
```
Comment (latedude2) [inline Assets/Scripts/HoldObject.cs]:
```suggestion
        layer = 1 << 8;  //we use the item layer or sth
        if (Physics.Raycast(transform.position, transform.forward, out RaycastHit hit, interactionDistance, layer))
```

**Review by latedude2 — CHANGES_REQUESTED** [review]

File: Assets/Scripts/HoldObject.cs (hunk 1 above)
Comment (Jokubas126) [reply Assets/Scripts/HoldObject.cs]: ah maybe 👍 I had it set up a little different when I assigned different keys. Will change that

File: Assets/Scripts/HoldObject.cs (hunk 2 above)
Comment (Jokubas126) [reply Assets/Scripts/HoldObject.cs]: Sure 👍

**Review by latedude2 — APPROVED** [review]

**Pull request merged**